src/components/modal/modal.css

```diff
@@ -0,0 +1,25 @@
+.modal_container {
+	display: none;
+	position: fixed;
+	top: 0;
+	right: 0;
+	width: 100vw;
+	height: 100vh;
+	align-items: center;
+	justify-content: center;
+	backdrop-filter: blur(1px);
+	z-index: 2;
+}
+
+.modal {
+	border-radius: 4px;
+	border: 1px solid var(--COLOR-BLACK);
+	background-color: var(--COLOR-BG);
+	max-width: 350px;
```

Comment:
You can consider increasing width slightly. It will look good. Also please do it in rems so that it does not look bad on small screens.

Reply:
Ok. I have blurred background more. Thanks for mentioning this.